Source/Resources/en.lproj/Localizable.strings

```diff
@@ -16,6 +16,8 @@
 "ExpiryDateFormat" = "Format is MM/YY";
 "ExpiryDatePlaceholder" = "MM/YY";
 "ExpiryDateErrorMessage" = "Missing Expiry Date";
+"CardNumber" = "Card Number";
+"CardNumberErrorMessage" = "Missing Card Number";
```

Comment:
Is this necessarily true? Would this error also get shown if the card number was incomplete or invalid?